### ISSUE TITLE
Boots of lightness weight reduction stack

### ISSUE THREAD

**Issue by RedSparr0w** [dginovker/2006rebotted#218]
If you equip boots of lightness multiple times, your weight will be reduced each time without increasing.

**Comment by dginovker** [comment]
From Dark98:

I'm not with my pc but for boots of lightness,under here
https://github.com/dginovker/2006rebotted/blob/d876a923b9d14410b4d8b860239ea4462dd6243a/2006Redone%20Server/src/com/rebotted/game/items/ItemAssistant.java#L1739
```
if (c.playerEquipment[slot] == 88) {    
  player.weight -= 4.5;            player.getPacketSender().writeWeight((int) player.weight);
}
```

**Comment by mrextremez** [comment]
I will fix when I get time if someone else hasn’t already

**Comment by Gptaqbc** [comment]
Fixed.

It wasn't exactly as Dark98 said but it was a really good starting point!

**Comment by RedSparr0w** [comment]
It doesn't happen anymore if you keep equiping **just** the boots of lightness,
But if you equip a different pair of boots, then boots of lightness alternating, then it still stacks.

**Comment by RedSparr0w** [comment]
Fixed by #272
